AutoscoperM/AutoscoperM.py

```diff
@@ -662,6 +670,7 @@ def onLoadPV(self):
                     break
             if vol not in paired:
                 logging.error(f"No transform found for {vol}")
+                return
```

Comment:
If there is an error, I suggest to exit and avoid having incomplete data loaded.